addons/mail/static/src/models/thread.js

```diff
@@ -1015,7 +1013,7 @@ registerModel({
             if (immediateNotify) {
                 this.throttleNotifyCurrentPartnerTypingStatus.clear();
             }
-            this.throttleNotifyCurrentPartnerTypingStatus({ isTyping: false });
+            this.throttleNotifyCurrentPartnerTypingStatus.do({ isTyping: false });
```

Comment:
Move `isTyping` param to field

Reply:
Note that is field already "exists", in the sense that existence of `currentPartnerInactiveTypingTimer` and/or `currentPartnerLongTypingTimer` already means `isTyping`.

Reply:
There is causality between typing and the (typing timers)... but I would be very cautious not using the timer(s) for deducing typing state.

Timer(s) are things that happen in reaction to (instantaneous) changes of typing boolean state.

It feels very off to do the opposite, because timers are there for relatively _**soft**_ design choices, whereas typing boolean state are _**hard**_ design choices. By "soft" vs "hard", I mean typing features will always need and keep boolean states, while timers can be added/removed/adjusted/overhauled while keeping the typing feature.